.github/workflows/prerelease_linux.yml

```diff
@@ -18,8 +18,8 @@ env:
   DOCKER_HUB_ID: ${{ secrets.OHAI_DOCKER_HUB_ID }}
   DOCKER_HUB_PASSWORD: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
   SCHEMA_BRANCH: ${{ github.event.release.tag_name }}
-  AWS_S3_BUCKET_NAME: "nr-downloads-ohai-staging"
-  AWS_S3_LOCK_BUCKET_NAME: "onhost-ci-lock-staging"
+  AWS_S3_BUCKET_NAME: "nr-downloads-ohai-testing"
```

Comment:
we will need to revert this change (it is getting annoying, we need to find better way with 1 flag)